src/pages/Home/NavBar/SearchBox/SearchBoxStyle.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
```

Comment:
Rename file to SearchBox.style.ts. You can distinguish style files by using the <>.style.ts way. So the of the component comes before style.ts